tests/migrations/test_commands.py

```diff
@@ -1870,6 +1872,64 @@ class Meta:
                 with self.assertRaises(SystemExit):
                     call_command('makemigrations', 'migrations', interactive=True)
 
+    @mock.patch('builtins.input', return_value='1')
+    def test_makemigrations_unique_callable_default_addition_interactive(self, mock_input):
+        """
+        makemigrations prompts the user when adding a unique field with
+        a callable default.
+        """
+        class Book(models.Model):
+            created = models.DateTimeField(unique=True, default=timezone.now)
+
+            class Meta:
+                app_label = 'migrations'
+
+        version = get_docs_version()
+        input_msg = (
+            'Callable defaults will not generate unique values upon migrating. '
+            'Found callable default on unique field book.created.\n'
```

Comment:
`book` should be `Book` ideally, I think?

I wonder also if the `Found ...` sentence should first. Telling the user about callable defaults without having told them what was found seems stiff. Though if it's following the cadence of other prompts (I can't recall), then disregard; better to be consistent.

Reply:
Good ear! I sort of adapted the existing cadence and made it worse by making it two complete sentences. I'll spruce up into one.

Reply:
About caps, yah spotted that also, but I noticed status quo, see:

https://github.com/django/django/blob/604df4e0adc71da264f61fe85020a170c98e6f09/tests/migrations/test_commands.py#L1375-L1378

Do you think we should open an easy pickings ticket to capitalize the model in each affected prompt?

Reply:
I don't think it's worth changing :shrug: